test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -76,7 +75,6 @@ public void LightGBMBinaryEstimatorUnbalanced()
             var trainer = ML.BinaryClassification.Trainers.LightGbm(new LightGbmBinaryTrainer.Options
             {
                 NumberOfLeaves = 10,
-                NumberOfThreads = 1,
                 MinimumExampleCountPerLeaf = 2,
```

Comment:
Can we add a detailed note somewhere in code visibly about why this is done? 
Can we also add a cross reference to that comment wherever we have removed the NumberOfThreads = 1

Reply:
Yes, I can add some comments with // Attention tags and put explain on comments

---
In reply to: [388442580](https://github.com/dotnet/machinelearning/pull/4918#discussion_r388442580) [](ancestors = 388442580)